cli/cmd/deploy/deploy.go

```diff
@@ -74,8 +79,8 @@ func DeployCmd(cfg *config.Config) *cobra.Command {
 			}()
 
 			// Verify that the projectPath contains a Rill project
-			if !rillv1beta.HasRillProject(projectPath) {
-				fullpath, err := filepath.Abs(projectPath)
+			if !rillv1beta.HasRillProject(fullProjectPath) {
```

Comment:
`fullProjectPatgh` is also required in `repoInSyncFlow`. We don't need to check for uncommitted changes in full project